.github/workflows/push.yml

```diff
@@ -35,9 +50,9 @@ jobs:
         echo ::set-output name=version::$version
         echo "DOCKER_TAG=${version}" >> $GITHUB_ENV
     - name: Build docker image 
```

Comment:
the comment at Line 36 is not relevant, we don't create helm charts here

Reply:
The `DOCKER_TAG` env is used later in the makefile as the tag of the docker image that is build and published, so I think that without this line the tag will always be `main` as the default value from the makefile.

Reply:
I meant the comment about helm